### PR TITLE
feat: Add logging and metrics

### PR DESCRIPTION
**Pull request by Elyniss**
Grafana panels will be added in next PR

### CODE REVIEW

File: .github/actions/install-rust/action.yml [hunk 1]
```diff
@@ -27,3 +27,6 @@ runs:
         profile: ${{ inputs.profile }}
         components: ${{ inputs.components }}
         override: true
+    
+    - name: Install Protoc
+      uses: arduino/setup-protoc@v1
```
Comment (arein) [inline .github/actions/install-rust/action.yml:32]: why?

Comment (arein) [reply .github/actions/install-rust/action.yml:32]: We removed this in other repos

File: src/handlers/invite/unregister.rs [hunk 2]
```diff
@@ -34,22 +36,60 @@ pub async fn handler(
...
 
     let claims: InviteKeyClaims = jwt.claims;
     let account = extract_did_data(&claims.pkh, DID_METHOD_PKH)?;
 
     let params = UnregisterInviteKeyParams {
         account: account.to_string(),
     };
-    params.validate()?;
+    params.validate().map_err(|error| {
+        info!(
```
Comment (arein) [inline src/handlers/invite/unregister.rs:69]:
```suggestion
        warn!(
```

File: .github/actions/install-rust/action.yml (hunk 1 above)
Comment (Elyniss) [reply .github/actions/install-rust/action.yml]: 🤔  It's required by `opentelemetry-prometheus` package

**Review by xav — APPROVED** [review]

**Pull request merged**